docs/refs/api/drivers/DigitalInOut.md

```diff
@@ -2,11 +2,14 @@
 
 Use the DigitalInOut interface as a bidirectional digital pin:
 
-* Read the value of a digital pin when set as an input.
-* Write the value when set as an output.
+* Read the value of a digital pin when set as an `input()`.
+* Write the value when set as an `output()`.
 
 You can use any of the numbered Arm Mbed pins as a DigitalInOut.
 
+**Tips:**
```

Comment:
Because this is only one tip, please delete the "s" from the header.

Reply:
done.